mlflow/pyfunc/backend.py

```diff
@@ -215,6 +217,12 @@ def predict(
                 )
             scoring_server._predict(local_uri, input_path, output_path, content_type)
 
+    @staticmethod
```

Comment:
Does this need to be a staticmethod here? Why not a normal internal function?

Reply:
can be a normal internal function.